packages/nx/src/daemon/client/client.ts

```diff
@@ -574,8 +573,8 @@ export class DaemonClient {
   }
 
   async startInBackground(): Promise<ChildProcess['pid']> {
-    ensureDirSync(DAEMON_DIR_FOR_CURRENT_WORKSPACE);
-    ensureFileSync(DAEMON_OUTPUT_LOG_FILE);
+    mkdirSync(DAEMON_DIR_FOR_CURRENT_WORKSPACE, { recursive: true });
+    writeFileSync(DAEMON_OUTPUT_LOG_FILE, '');
```

Comment:
This doesn't seem to be equivalent to ensureFileSync: https://github.com/jprichardson/node-fs-extra/blob/d96f2655f181f1c6b1641711619400140a89d027/lib/ensure/file.js#L40

It won't write to the file if it already exists

Reply:
Good catch! Should be good now. I think my simple check is sufficient in this case because we're calling `mkdirSync` before, therefore the parent directory should always exist.